schema/src/main/java/com/criteo/hadoop/garmadon/protocol/ProtocolVersion.java

```diff
@@ -4,7 +4,7 @@
 import java.util.List;
 
 public class ProtocolVersion {
-    public static final byte VERSION = 1;
+    public static final byte VERSION = 3;
```

Comment:
Seems like this is a hack to prevent authorizing this release. Shouldn't we instead manager the AUTHORIZED release by hand and explicitly list still managed versions ?

Reply:
I don't think that managing release manually is a good  choice

Reply:
How is it worse than a piece of code forcing to accept the previous version and hacking it the first time we use it ?

Reply:
To add clarity on "managing by hand", I was meaning to populate the list line 10 with actual version supported and not necessarily the previous one.